README.md

```diff
@@ -317,13 +319,20 @@ or "disabled" (to disable the Process Agent entirely)
 - `scrub_args` - enables the scrubbing of sensitive arguments from a process command line. Default value is true
 - `custom_sensitive_words` - expands the default list of sensitive words used by the cmdline scrubber
 
+#### System Probe
+
+The network performance monitoring system probe is configured under the `system_probe_config` variable.  Any variables nested underneath will be written to the `system-probe.yaml`
```

Comment:
Could you add a link to the official system-probe doc page. Also please mention that this is only supported for Agent6 Linux.